haystack/pipeline.py

```diff
@@ -291,6 +291,40 @@ def _overwrite_with_env_variables(cls, definition: dict):
                 param_name = key.replace(env_prefix, "").lower()
                 definition["params"][param_name] = value
 
+    def export_yaml_config(self, path: Path):
```

Comment:
Let's rename the method to `save_to_yaml` to have consistent naming with the equivalent `load_from_yaml()`.